SPECS/patchelf/patchelf.spec

```diff
@@ -1,25 +1,22 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-# hardening breaks the set-interpreter-long test on i686, x86_64, ppc64le, s390x
-%undefine _hardened_build
-
+Summary:        A utility for patching ELF binaries
 Name:           patchelf
 Version:        0.12
 Release:        3%{?dist}
-Summary:        A utility for patching ELF binaries
-
 License:        GPLv3+
-URL:            http://nixos.org/patchelf.html
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://nixos.org/patchelf.html
 Source0:        https://github.com/NixOS/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
-
-BuildRequires:  gcc
-BuildRequires:  gcc-c++
-BuildRequires:  make
 BuildRequires:  autoconf
 BuildRequires:  automake
 BuildRequires:  coreutils
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
 BuildRequires:  libacl-devel
 BuildRequires:  libattr-devel
+BuildRequires:  make
+# hardening breaks the set-interpreter-long test on i686, x86_64, ppc64le, s390x
+%global _hardened_build 1
```

Comment:
Nit: this is enabled by default, so you can even remove the macro. I'd leave the comment and find a fix (or disable) for that one test failing for x86_64.